### ISSUE TITLE
TFT35 E3 V3 firmware does not work

### ISSUE THREAD

**Issue by RocksaltX** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1912]
Send firmware to TFT, screen is locked up.

### Steps to reproduce

1. Copy BIGTREE_TFT35_V3.0_E3.27.x.bin to flash card, copy TFT folder, copy config.ini
2. boot
3. Frown

**Expected behavior**
Work,,,
**Actual behavior**
Screen froze, can see BTT logo in the status area, touch does not work, holding button in for many seconds does not bring up marlin mode.

### Hardware Variant

TFT35 E3 V3

BIGTREE_TFT35_V3.0_E3.27.x.bin, Marlin 2.0 Bug may 2011

### Additional Information
Reverting back to a .zip I downloaded on 4/4/21 works properly.
I did reset the TFT in the menu and tried it again with same issue.

**Comment by kisslorand** [comment]
Did you try to reset the TFT touch calibration with "reset.txt" file placed in the root of your SD card?

**Comment by RocksaltX** [comment]
yes, I did. That just resets the touch calibration. Even when pressing the rotary dial to get to marlin mode it does not work. It's locked up with this .bin file. As I stated, if I revert back to a saved version it's working.

**Comment by oldman4U** [comment]
Is this master or v27 release version? Self compiled or pre compiled?

**Comment by oldman4U** [comment]
All your previous postings are about TFT35 v2. Have you changed the TFT hardware to a new TFT35 E3 V3?

**Comment by oldman4U** [comment]
Just updated 2 TFT35 E3 v3 with the precompiled version from today without any issue.

Please help the community and close the ticket once you do not need it anymore. Thank you

**Comment by RocksaltX** [comment]
Well shoot-fire. Sorry to waste your time. I downloaded the version today and went at it again. Same result. I made a mistake and did not copy the TFT folder, at least it seems that way. I loaded the new firmware and config.ini and had the same problem. I then did it again with the TFT folder and it worked. I am 99% sure I loaded the TFT folder when I opened this request or when I downloaded the TFT files on 5/11/21.
Again, I apologize.
